### PR TITLE
Examples review

### PR DESCRIPTION
**Pull request by mattiasostmar**
Fixes based on feedback caught after upload of 10 example images described in [Phabrikator Task T144358](https://phabricator.wikimedia.org/T144358#2867555).

### CODE REVIEW

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: Should be {{creator:Sigvald_Linné}}{{creator:Ola Apenes}}

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: Don't think this `if` is needed since it can never be true (since `row["Personnamn / fotograf"]`) is `null` per the `if` before.

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py:335]: This should be reverted. `/` is correct here (the note in [T144358#2867555 ](https://phabricator.wikimedia.org/T144358#2867555) was about the photographer field above)

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: The old logic was correct but likely failing due to the accent on the "e".

I would revert and instead try just changing `if j + " " + i in ("Sigvald Linne", "Sigvald Linné")`.

Two changes though:
* There should be a `/` after `[[:d:Q5959424|Sigvald Linné]]` not a `\n`
* don't set `linne_category = True` since this is what you use to indicate that he was the photographer

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py:350]: One space should be removed here as well

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: To differentiate the content category (images of Linné) and the meta category (images by Linné) you could change this one to `[[Category:Images_by_Sigvald_Linné]]`.

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: also should set `linne_category = True`

**Review by lokal-profil — CHANGES_REQUESTED** [review]
I made a few comments. Please address these then ping me for a second review before merging.

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: Since the following check comes afterwards, doesn't it suffice with adding {{creator:Ola_Apenes}}?

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: Fixed in 76d6b694caee3734468ca4ae6ba0447cc82ab3cc

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: See fix ee1acc7cf7de437dcea76fb96f1fd035f0bacad0 that is my suggested solution. Also did you solution in previous commit 9ddd7377ac5b42a862b234dfb97310e70680a0d3

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: 1. Fix accent issue fix: 54ae3ec47f0b6d9680d8f4b23063921cd3a5dda2
2. Fix Linné in multiple depicted people list: 68a9f8fc8eb8d589268f68eb289289e88cfe4fba
3. Fix no linne_category: 7e45e5eb17cb8f96a2617a29d13279f69344b174

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: Fix: fe99b4b14d70958487de8890640cf2f7827da08c

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py:335]: Fixed the / in 267861c34fe1161c475f962e62d2c6d0d539a91f

Do you also really mean I should delete lines 331, 332, 333?

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py:350]: Fixed in ab5d176c657a75feed9d4d5239058c863ce7016f

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: Fix: 2e7045e18f59b6c874d32c7e246b152e8bbe8df1

**Review by mattiasostmar — COMMENTED** [review]
Answered all (even though github says only for) of your comments wih'th fixes. Have a look.

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: To clarify. What I meant was replacing everything from (the current) line 327 downwards by:
```
         else:
             for i, j in zip(lista[::2], lista[1::2]):
                if j + " " + i == ("Sigvald Linné", "Sigvald Linne"):
                    depicted_people += "[[:d:Q5959424|Sigvald Linné]]/"
                    content_categories_string += "[[Category:Sigvald_Linné]]\n"
                else:
                    depicted_people += j + " " + i + "/"
```
That is what you had originally was good apart from the accent and the linne_category both of which are fixed in the above code.

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (lokal-profil) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py:335]: Se comment above (with example code) =)

File: 1. Mexiko Create filenames filename-mappings and infofiles.py
Comment (mattiasostmar) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: Ah, OK. 
Fixed with a slight modification - 'in' instead of '=' in a68277ccd69763813c1561b96874e8eff9297f17

**Review by lokal-profil — APPROVED** [review]
Looks good. Use "Squash and merge" from the merge button dropdown when merging.

**Pull request merged**
